pkg/controller/logstash/logstash_controller.go

```diff
@@ -113,7 +117,8 @@ type ReconcileLogstash struct {
 	dynamicWatches watches.DynamicWatches
 	operator.Parameters
 	// iteration is the number of times this controller has run its Reconcile method
-	iteration uint64
+	iteration    uint64
+	expectations *expectations.ClustersExpectation
```

Comment:
We used the [ClustersExpectation](https://github.com/elastic/cloud-on-k8s/blob/main/pkg/controller/common/expectations/per_cluster.go), rather than the standard `Expectation` to allow expectations for multiple sets of Logstashes to be stored safely in the same ECK instance, and my understanding was that this was required

Reply:
Yes my point was more about the naming, because Logstash is not really a clustered application,  not about the fact that you are using it which is appropriate.